From 00a5736ff2220229d75d4259c582ae0a62539915 Mon Sep 17 00:00:00 2001
From: Jim Meyering <meyering@redhat.com>
Date: Tue, 28 Aug 2012 19:49:25 +0200
Subject: [PATCH 08/89] maint: don't prohibit strncpy just yet

* cfg.mk (local-checks-to-skip): Add sc_prohibit_strncpy,
so that we do not yet enable the strncpy prohibition.
There are many uses, and it will be a lot of work to remove
all of them.
---
 cfg.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cfg.mk b/cfg.mk
index 45d2ac2..3d3014c 100644
--- a/cfg.mk
+++ b/cfg.mk
@@ -34,6 +34,7 @@ local-checks-to-skip = \
   sc_error_message_period \
   sc_file_system \
   sc_prohibit_strcmp \
+  sc_prohibit_strncpy \
   sc_prohibit_atoi_atof \
   sc_require_test_exit_idiom \
   sc_space_tab \
-- 
1.8.5.3

